src/main/resources/templates/entries.html

```diff
@@ -3,74 +3,76 @@
+            <ol>
+                <li>
+                    <a href="/" th:utext="${friendlyRegisterName}">Home</a>
+                </li>
+                <li th:if="${content.recordKey.present}">
+                    <a href="/records">Records</a>
+                </li>
+                <li th:if="${content.recordKey.present}">
+                    <a th:href="${'/record/' + content.recordKey.get()}"
+                       th:text="${'Record ' + content.recordKey.get()}"></a>
+                </li>
+                <li>Entries</li>
+            </ol>
+        </div>
 
-      <div class="grid-row">
-        <div class="column-two-thirds">
-          <h1 class="heading-large">
+        <div class="grid-row">
+            <div class="column-two-thirds">
+                <h1 class="heading-large">
             <span th:if="${content.recordKey.present}">
               Entries for record <span th:text="${content.recordKey.get()}">GM</span>
             </span>
-            <span th:unless="${content.recordKey.present}">
+                    <span th:unless="${content.recordKey.present}">
               Entries
             </span>
-            <span class="heading-secondary" th:include="fragments/pagination.html::pagination-top('entries')"></span>
-          </h1>
-          <div th:include="fragments/data-formats.html::data-formats"></div>
+                    <span class="heading-secondary"
+                          th:include="fragments/pagination.html::pagination-top('entries')"></span>
+                </h1>
+                <div th:include="fragments/data-formats.html::data-formats"></div>
+            </div>
+            <div class="column-third" th:include="fragments/attribution.html::attribution"></div>
         </div>
-        <div class="column-third" th:include="fragments/attribution.html::attribution"></div>
-      </div>
 
-      <div class="table-wrapper">
-        <table th:if="${#lists.isEmpty(content.entries) == false}">
-          <thead>
-          <tr>
-            <th scope="col">Entry</th>
-            <th scope="col">item-hash</th>
-            <th scope="col">entry-timestamp</th>
-            <th scope="col">key</th>
-          </tr>
-          </thead>
-          <tbody>
-          <tr th:each="entry : ${content.entries}">
-            <td>
-                <a th:href="${'/entry/' + entry.entryNumber}" th:text="${entry.entryNumber}"></a>
-              </td>
-            <td>
-              <a th:href="${'/item/' + entry.sha256hex}" th:text="${entry.sha256hex}"></a>
-            </td>
-            <td th:text="${entry.timestamp}"></td>
-            <td th:text="${entry.key}"></td>
-          </tr>
-          </tbody>
-        </table>
-      </div>
+        <div class="table-wrapper">
+            <table th:if="${#lists.isEmpty(content.entries) == false}">
+                <thead>
+                <tr>
+                    <th scope="col">Entry</th>
+                    <th scope="col">item-hashes</th>
```

Comment:
This column name needs to remain `item-hash` (and also same for entry-table.html)